parachain/scripts/clean-network.sh

```diff
@@ -19,15 +19,15 @@ esac
 ZOMBIENET_BIN=zombienet-${os}-${arch}
 ZOMBIENET_PID=$(pidof $ZOMBIENET_BIN)
 
-if [ -z $ZOMBIENET_PID ]; then
+if [ -z "$ZOMBIENET_PID" ]; then
   # the network might not be started with zombienet
   killall polkadot || true
   killall litentry-collator || true
 else
-  kill -2 $ZOMBIENET_PID
+  sudo kill -2 $ZOMBIENET_PID
```

Comment:
I think we don't need sudo here

Reply:
wired. I was getting something like `./scripts/clean-network.sh: line 27: kill: (388645) - Operation not permitted`. 
But now it's disappeared. 
I will remove this `sudo`.